### ISSUE TITLE
Errata: Suggestions for HHB text

### ISSUE THREAD

**Issue by orjanbp** [Jihia/Warcraft-5e-Conversion#314]
>The druid flavor text has some odd sentence structure, as well as some typos
> - “smashing upon its body and encasing it hole”
> - “towards the constructions of Zul’Gurub. Watching every move of the jungle trolls patrolling”
> - “Holding near her sickle and staff, a young tauren druid is making beautiful ritual, making tiny seeds sprinkle...”
> - “Rushing forward...with its fur”

> First paragraph in chapter 1: maybe rephrase “Most players belong to one or the other faction” to “Most players belong to one faction or the other”
> - Maybe its just nitpicky, but roleplay is typically 1 word in official D&D books (see chapter 1, 2nd paragraph of “choosing a side”)
> - “Diplomacy is key with in the Alliance” (typo, should be within - first paragraph of “The Alliance” in chapter 1)
> - Mountains in Redridge Mountains should be capitalized if Elwynn Forest is capitalized (2nd paragraph of “The Alliance, chapter 1)
> - “...and the Redridge mountains. As well as having outposts and bases stationed...” period should be semicolon or comma (2nd paragraph of “The Alliance”, chapter 1)
> - “The majority of Alliance citizens also recognizes” should be recognize (1st paragraph of “The Alliance”, chapter 1)
> - I’m not sure if it should be “the Ashenvale” or just “Ashenvale” (2nd paragraph of “The Alliance”, chapter 1)

**Comment by Jihia** [comment]
https://docs.google.com/document/d/1Vm_CUe88mdT0845ws7QYt_AJipB5ympg0uAWdfmeX74/edit

Additional suggestions and errors for us to look at has been compiled in this document.
